covalent/_file_transfer/strategies/__init__.py

```diff
@@ -20,3 +20,10 @@
 
 from .http_strategy import HTTP
 from .rsync_strategy import Rsync
+
+try:
+    import boto3
+
+    from .s3_strategy import S3
+except:
+    print("To use the S3 file transfer strategy kindly install the boto3 packagefrom AWS")
```

Comment:
Shouldnt this be inside S3 init class? This would except when boto3 is not present while importing any strategy, not just S3.

Reply:
The difference is whether the strategy even comes up as an option.  In one case, it's not part of the namespace and the module isn't available; in the other, it's available but throws an error upon instantiation.  Probably we should put the error message in both places, in case the file is explicitly used.

Reply:
I have removed the raise except in __init__.py and added it inside the s3 init. I hope this has addressed the issue/concerns.

Reply:
Done